test/node-rpc-test.js

```diff
@@ -68,11 +70,75 @@ describe('RPC', function() {
     assert.deepEqual(info.localservicenames, ['NETWORK', 'WITNESS']);
   });
 
+  it('should rpc getblockhash', async () => {
+    const info = await nclient.execute('getblockhash', [node.chain.tip.height]);
+    assert.strictEqual(util.revHex(node.chain.tip.hash), info);
+  });
+
+  describe('Blockchain', function () {
+    it('should rpc getchaintips', async () => {
+      const info = await nclient.execute('getchaintips', []);
+      assert.strictEqual(info.length, 1);
+      assert.strictEqual(util.revHex(node.chain.tip.hash), info[0].hash);
+    });
+
+    it('should rpc getchaintips when a block is mined', async () => {
+      const entry = await node.chain.getEntry(node.chain.tip.height - 1);
+      const block = await node.miner.mineBlock(entry);
+      await node.chain.add(block);
+
+      const info = await nclient.execute('getchaintips', []);
+      assert.strictEqual(info.length, 1);
+      assert.strictEqual(util.revHex(node.chain.tip.hash), info[0].hash);
```

Comment:
```suggestion
      assert.strictEqual(util.revHex(util.revHex(entry.hash)), info[0].hash);
```

I think this way it is super-duper clear that the block you mined and added on line 86 is the current chain tip and is returned by the rpc call.